app/docker/components/datatables/containers-datatable/containersDatatable.html

```diff
@@ -105,45 +156,48 @@
                   </div>
                 </div>
               </th>
-              <th ng-if="$ctrl.settings.showQuickActionStats || $ctrl.settings.showQuickActionLogs || $ctrl.settings.showQuickActionConsole || $ctrl.settings.showQuickActionInspect">
+              <th ng-show="$ctrl.columnVisibility.columns.created.display">
```

Comment:
Please add this column after the Image column.

Also, please add the ability to sort this column as it's done for other columns.